angularjs-portal-mock-portal/src/main/webapp/web/layoutDoc

```diff
@@ -69,7 +69,7 @@
          "target":null,
          "widgetURL":"/portal/p/earnings-statement/max/earningStatements.resource.uP",
          "widgetType":"generic",
-         "widgetTemplate":"\n        <div style='margin : 0 10px 0 10px;'>\n           <loading-gif data-object='content' data-empty='isEmpty'></loading-gif>\n           <ul class='widget-list'><li ng-repeat='item in content.report |orderBy: [\"-paid.substring(6)\",\"-paid.substring(0,2)\",\"-paid.substring(3,5)\"] | limitTo:3' class='center'><a href='/portal/p/earnings-statement/max/earning_statement.pdf.resource.uP?pP_docId={{item.docId}}' target='_blank'><i class='fa fa-bank fa-fw'></i> {{item.paid}} Statement</a></li></ul>\n           <div ng-if='isEmpty' style='padding: 10px; font-size: 14px;'><i class='fa fa-exclamation-triangle fa-3x pull-left' style='color: #b70101;'></i><span style='color: #898989;'>We had a problem finding your statements (or you don't have any).</span></div>\n           <div style='background-color: #EAEAEA; border-radius:4px;padding:10px; margin-top:10px;'>\n              <span class='bold display-block left' style='text-align: left; padding-left: 10px; font-size: 14px;'>See all payroll information for more options:</span> \n              <ul style='text-align: left;list-style-type: disc; font-size: 12px;'>\n                 <li>See all paystubs</li>\n                 <li>Tax statements</li>\n                 <li>Update direct deposit</li>\n              </ul>\n           </div>\n        </div>\n        <a class='btn btn-default launch-app-button' href='/portal/p/earnings-statement'>See all payroll information</a>\n        ",
+         "widgetTemplate":"\n <div style='margin : 0 10px 0 10px;'>\n <loading-gif data-object='content' data-empty='isEmpty'></loading-gif>\n <ul class='widget-list'><li ng-repeat='item in content.report |orderBy: [\"-paid.substring(6)\",\"-paid.substring(0,2)\",\"-paid.substring(3,5)\"] | limitTo:3' class='center'><a href='/portal/p/earnings-statement/max/earning_statement.pdf.resource.uP?pP_docId={{item.docId}}' target='_blank'> {{item.paid}}Earnings Statement</a></li></ul>\n <div ng-if='isEmpty' style='padding: 10px; font-size: 14px;'><i class='fa fa-exclamation-triangle fa-3x pull-left' style='color: #b70101;'></i><span style='color: #898989;'>We had a problem finding your statements (or you don't have any).</span></div>\n <div style='background-color: #EAEAEA; border-radius:4px;padding:10px; margin-top:10px;'>\n <span class='bold display-block left' style='text-align: left; padding-left: 10px; font-size: 14px;'>Launch payroll to find::</span> \n <ul style='text-align: left;list-style-type: disc; font-size: 12px;'>\n <li>Earnings statements</li>\n <li>Tax statements (W-2, 1095-C, etc.)</li>\n <li>W-4 and direct deposit forms</li>\n </ul>\n </div>\n </div>\n <a class='btn btn-default launch-app-button' href='/portal/p/earnings-statement'>See all payroll information</a>\n         ",

```

Comment:
`{{item.paid}}Earnings Statement</a>`

I think our style guidance would have this camel case, as in

`{{item.paid}}Earnings statement</a>`

Do I have that style guidance right?